File: app/src/main/java/com/nckh2016/vuduytung/nckh2016/object/ObjectCTDT.java
package com.nckh2016.vuduytung.nckh2016.object;

/**
 * Created by Tung on 3/3/2016.
 */
public class ObjectCTDT implements Items {
    String mabm;
    String mamh;
    int hocky;
    int chuyennganh;
    String tuchon;

    String tenmh;
    String tinchi;

    @Override
    public boolean isHocKy() {
        return false;
    }

    public ObjectCTDT(String mabm, String mamh, int hocky, int chuyennganh, String tuchon, String tenmh, String tinchi) {
        this.mabm = mabm;
        this.mamh = mamh;
        this.hocky = hocky;
        this.chuyennganh = chuyennganh;
        this.tuchon = tuchon;
        this.tenmh = tenmh;
        this.tinchi = tinchi;
    }

    public String getMabm() {
        return mabm;
    }

    public void setMabm(String mabm) {
        this.mabm = mabm;
    }

    public String getMamh() {
        return mamh;
    }

    public void setMamh(String mamh) {
        this.mamh = mamh;
    }

    public int getHocky() {
        return hocky;
    }

    public void setHocky(int hocky) {
        this.hocky = hocky;
    }

    public int getChuyennganh() {
        return chuyennganh;
    }

    public void setChuyennganh(int chuyennganh) {
        this.chuyennganh = chuyennganh;
    }

    public String getTuchon() {
        return tuchon;
    }

    public void setTuchon(String tuchon) {
        this.tuchon = tuchon;
    }

    public String getTenmh() {
        return tenmh;
    }

    public void setTenmh(String tenmh) {
        this.tenmh = tenmh;
    }

    public String getTinchi() {
        return tinchi;
    }

    public void setTinchi(String tinchi) {
        this.tinchi = tinchi;
    }
}

File: WinForm_CapNhatCTDT/WinForm_CapNhatCTDT/LogicLayer/ObjectCTDT.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinForm_CapNhatCTDT.LogicLayer
{
    class ObjectCTDT
    {
        String mabm;
        String mamh;
        Int64 hocky;
        Int64 chuyennganh;
        String tuchon;
        String tenmh;
        String tinchi;

        public String MaBoMon { get { return mabm; } set { mabm = value; } }
        public String MaMonHoc { get { return mamh; } set { mamh = value; } }
        public Int64 HocKy { get { return hocky; } set { hocky = value; } }
        public Int64 ChuyenNganh { get { return chuyennganh; } set { chuyennganh = value; } }
        public String TuChon { get { return tuchon; } set { tuchon = value; } }
        public String TenMonHoc { get { return tenmh; } set { tenmh = value; } }
        public String TinChi { get { return tinchi; } set { tinchi = value; } }

        public ObjectCTDT(String mabm, String mamh, Int64 hocky, Int64 chuyennganh, String tuchon, String tenmh, String tinchi)
        {
            this.mabm = mabm;
            this.mamh = mamh;
            this.hocky = hocky;
            this.chuyennganh = chuyennganh;
            this.tuchon = tuchon;
            this.tenmh = tenmh;
            this.tinchi = tinchi;
        }
    }
}
